### PR TITLE
Migrate ovirt specs to gunicorn and apache

### PR DESCRIPTION
**Pull request by mnecas**
There is diffrence between spec files from which we are building rpms and which are in this repo.
@mwperina

### CODE REVIEW

File: packaging/gunicorn/ansible-runner-service.el7.spec [hunk 1]
```diff
@@ -0,0 +1,105 @@
+%global srcname ansible-runner-service-dev
```
Comment (mwperina) [inline packaging/gunicorn/ansible-runner-service.el7.spec:1]: EL7 is no longer supported platform for oVirt 4.4, so what do we need to have this spec for?

File: packaging/gunicorn/ovirt_logging_path.patch [hunk 2]
```diff
@@ -0,0 +1,13 @@
+diff --git a/logging.yaml b/logging.yaml
+index 8357727..a883306 100644
+--- a/logging.yaml
++++ b/logging.yaml
+@@ -17,7 +17,7 @@ handlers:
+         class: logging.handlers.RotatingFileHandler
+         level: DEBUG
+         formatter: simple
+-        filename: ansible-runner-service.log
++        filename: /var/log/ovirt-engine/ansible-runner-service.log
```
Comment (mwperina) [inline packaging/gunicorn/ovirt_logging_path.patch]: Shouldn't we create /var/log/ovirt-engine directory as a part of -dev RPM installation?

Comment (mnecas) [reply packaging/gunicorn/ovirt_logging_path.patch]: That is older commit have migrated the patch to https://github.com/ansible/ansible-runner-service/pull/70

File: packaging/gunicorn/ansible-runner-service.el7.spec (hunk 1 above)
Comment (mnecas) [reply packaging/gunicorn/ansible-runner-service.el7.spec:1]: I wanted to save the python2 spec file but will remove it.

**Pull request merged**